### DISCUSSION TITLE
Nice Job !

### DISCUSSION THREAD

**Discussion by Soulman38** [salem-ok/PySprint#47] [General]
Hi Salem, 
C'est moi, ton binôme de l'époque de Super Sprint ;)
Just first words to congratulate for this playable version ! What a "return to the past" feeling from the moment I launch the .exe
So far, so good. I didn't notice any bugs yet. 
I don't recall there was a sliding effect when the car slow down ?
Will test further and provides more feedback.

**Comment by salem-ok** [comment]
Thanks Man! Much appreciated! 

You're totally right about the sliding I've got similar feedback from 2 other Super Sprint veterans...

 I think I am close from the original, but not quite right. I've got an issue open on it ([Issue 45](https://github.com/salem-ok/PySprint/issues/45)), feel free to comment and add you opinion, as we are working on "feeling" and perception only, since I am unable to decompile and read 68k ASM code form the original  :/
